<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIKanSoloMain</class>
 <widget class="QMainWindow" name="UIKanSoloMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kan Solo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QComboBox
{
	font-size:18px;
	background:#77ffffff;
	color:black;
	padding:4px;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="m_frmCommands">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="m_btnNewTask">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background: transparent;
	border:none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnNewBoard">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background: transparent;
	border:none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/imgBtnNewBoardSmall.png</normaloff>:/imgBtnNewBoardSmall.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnExit">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background: transparent;
	border:none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/imgBtnExitSmall.png</normaloff>:/imgBtnExitSmall.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QFrame" name="m_frmSelections">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	background:#77ffffff;
font-size:18px;
border:1px solid black;
border-right:none;
}</string>
           </property>
           <property name="text">
            <string>Current Board:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="m_cmbBoards">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="m_cmbThemes"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	background:#77ffffff;
font-size:18px;
border:1px solid black;
border-right:none;
}</string>
           </property>
           <property name="text">
            <string>Theme:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="m_frmBoard">
        <layout class="QVBoxLayout" name="m_boardLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_lblLogo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/imgLogoSmallDecorated.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
